test/e2e-test-application/cypress/e2e/tests/1-angular/luigi-client-link-manager-features-4.cy.js

```diff
@@ -1,16 +1,13 @@
 describe('Luigi Client linkManager Webcomponent, Drawer', () => {
-  beforeEach(() => {
-    cy.visitLoggedIn('/');
-  });
-
   describe('Drawer', () => {
     let $iframeBody;
+
     beforeEach(() => {
       cy.visitLoggedIn('/projects/pr2');
       cy.getIframeBody().then(result => {
         $iframeBody = result;
         cy.goToLinkManagerMethods($iframeBody);
-        cy.expectPathToBe('/projects/pr2');
+        // cy.expectPathToBe('/projects/pr2');
```

Comment:
was there a benefit to commenting this out or did it seem to be unnecessary anymore to check path 
If we don't need this at all here, and its better without, maybe remove it altogether. 🤔

Reply:
Very true, thanks. The reason for removing it was that `goToLinkManagerMethods` already does that assertion so there is no reason to do it again right after.